packages/cfd/src/Containers/cfd-password-modal/modal-elements/password-modal-message.tsx

```diff
@@ -0,0 +1,143 @@
+import {
+    getDerivezCompanies,
+    getDxCompanies,
+    getFormattedJurisdictionCode,
+    getMtCompanies,
+    TDerivezCompanies,
+    TDxCompanies,
+    TMtCompanies,
+} from '../../../Stores/Modules/CFD/Helpers/cfd-config';
+import { Localize } from '@deriv/translations';
+
+type TReviewMsgForMT5 = {
+    is_selected_mt5_verified: boolean;
+    jurisdiction_selected_shortcode: string;
+    manual_status: string;
+};
+
+type TPasswordModalMessage = TReviewMsgForMT5 & {
+    category: string;
+    is_wallet_enabled: boolean;
+    platform: string;
+    wallet_account_title: string;
+    show_eu_related_content: boolean;
+    type: string;
+};
+
+const ReviewMessageForMT5 = ({
+    is_selected_mt5_verified,
+    jurisdiction_selected_shortcode,
+    manual_status,
+}: TReviewMsgForMT5) => {
+    if (is_selected_mt5_verified) {
+        return (
+            <Localize i18n_default_text='To start trading, top-up funds from your Deriv account into this account.' />
+        );
+    } else if ([Jurisdiction.BVI, Jurisdiction.VANUATU].includes(jurisdiction_selected_shortcode)) {
+        if (manual_status === 'pending') {
+            return <Localize i18n_default_text='We’re reviewing your documents. This should take about 1 to 3 days.' />;
+        }
+        return <Localize i18n_default_text='We’re reviewing your documents. This should take about 5 minutes.' />;
+    } else if ([Jurisdiction.LABUAN, Jurisdiction.MALTA_INVEST].includes(jurisdiction_selected_shortcode)) {
+        return <Localize i18n_default_text='We’re reviewing your documents. This should take about 1 to 3 days.' />;
+    }
+    return null;
+};
+
+const PasswordModalMessage = ({
+    category,
+    is_selected_mt5_verified,
+    is_wallet_enabled,
+    jurisdiction_selected_shortcode,
+    manual_status,
+    platform,
+    show_eu_related_content,
+    type,
+    wallet_account_title,
+}: TPasswordModalMessage) => {
+    if (!category && !type) return null;
+
+    const category_label = getCategoryText(category);
+    let type_label = '';
+    switch (platform) {
+        case CFD_PLATFORMS.MT5:
+            type_label =
+                getMtCompanies(show_eu_related_content)[category as keyof TMtCompanies][
+                    type as keyof TMtCompanies['demo' | 'real']
+                ].short_title;
+            break;
+        case CFD_PLATFORMS.DXTRADE:
+            type_label =
+                getDxCompanies()[category as keyof TDxCompanies][type as keyof TDxCompanies['demo' | 'real']]
+                    .short_title;
+            break;
+        case CFD_PLATFORMS.DERIVEZ:
+            type_label =
+                getDerivezCompanies()[category as keyof TDerivezCompanies][
+                    type as keyof TDerivezCompanies['demo' | 'real']
+                ].short_title;
+            break;
+        default:
+            type_label = '';
+            break;
+    }
```

Comment:
Since type_label is not reassigned a new value wouldn't it be better to change it to a const and immediately declare their values i.e:
```suggestion
    const type_label = (() => {
        switch (platform) {
            case CFD_PLATFORMS.MT5:
                return getMtCompanies(show_eu_related_content)[category as keyof TMtCompanies][type as keyof TMtCompanies['demo' | 'real']].short_title;
            case CFD_PLATFORMS.DXTRADE:
                return getDxCompanies()[category as keyof TDxCompanies][type as keyof TDxCompanies['demo' | 'real']].short_title;
            case CFD_PLATFORMS.DERIVEZ:
                return getDerivezCompanies()[category as keyof TDerivezCompanies][type as keyof TDerivezCompanies['demo' | 'real']].short_title;
            default:
                return '';
        }
})();
```

Reply:
This is the same as before only in a form of function. I would suggest to keep it as it is because later on the CFD team will refactor it. 👍🏼